statsmodels/stats/multitest.py

```diff
@@ -264,49 +264,57 @@ def multipletests(pvals, alpha=0.05, method='hs', is_sorted=False,
 
 
 def fdrcorrection(pvals, alpha=0.05, method='indep', is_sorted=False):
-    '''pvalue correction for false discovery rate
+    '''pvalue correction for false discovery rate.
```

Comment:
I strongly prefer multi line docstrings to start on the next line, e.g.,

```
'''
Start of the docstring
```

but this is not universal.

Reply:
```suggestion
    '''
    pvalue correction for false discovery rate.
```